Core/wbDefinitionsCommon.pas

```diff
@@ -437,6 +518,76 @@ function wbNormalizeToRange(aMin, aMax: Extended): TwbFloatNormalizer;
 
 procedure DefineCommon;
 begin
+  wbRecordFlagsFlags := wbFlags(wbRecordFlagsFlags, [
+    {0x00000001} { 0} 'Unknown 0',
+    {0x00000002} { 1} 'Unknown 1',
+    {0x00000004} { 2} 'Unknown 2',
+    {0x00000008} { 3} 'Unknown 3',
+    {0x00000010} { 4} 'Unknown 4',
+    {0x00000020} { 4} 'Unknown 5',
```

Comment:
I understand it's commented out, but isn't { 4} supposed to be { 5} for 'Unknown 5'?

Reply:
heh that's been like that for who knows how long.  I just copypasta'd it over to Common.